lib/core/enums/swipe_action.dart

```diff
@@ -1 +1,15 @@
-enum SwipeAction { upvote, downvote, reply, save, edit, none }
+enum SwipeAction {
+  upvote(friendlyName: 'Upvote'),
```

Comment:
I would change this from `friendlyName` to `label` to make it more clear!

Reply:
Thanks, that's much better!